src/couch_jwt_auth.erl

```diff
@@ -58,6 +58,12 @@ decode(Token, Config) ->
       end, List), posix_time(calendar:universal_time()), Config)
   end.
 
+ensure_safe_token(Token, Config) ->
+  case couch_util:get_value(Token, Config) of
```

Comment:
Right, I thought about `/2` (:
For testing see https://github.com/apache/couchdb/blob/1.x.x/test/couchdb/couch_config_tests.erl 
TL;DR start couch_config app, put there some value, call your function.